### DISCUSSION TITLE
Satellite Macro Models for SimulateAccelerometer

### DISCUSSION THREAD

**Discussion by Namesume** [groops-devs/groops#141] [Q&A] [answered]
Dear Developers,

Hello, I'm simulating the accelerometer by using groops, to be honest, groops is a cool software for its applicability and convenience. However, I have a problem. You know, simulating the accelerometer needs satellite model (such as the GRACE macro model), and I've found some satellites' model such as swarm, grace(grace-fo), tandemX and terraSarX. 
Would you please tell me how can i get the satellite model of CHAMP, GOCE and Sentinel?

Like always thank you in advance for your time and help.

With kind regards,
Haonan Shi

**Comment by akvas** [comment, marked as answer]
Dear Haonan Shi,

we have some satellite macro models in the GROOPS format on our [FTP server](https://ftp.tugraz.at/outgoing/ITSG/groops/data/satelliteModel/). For other satellites you would need to consult either the mission documentation or research articles an generate these files yourself.

Cheers,
Andreas

**Reply by Namesume** [reply]
Dear akvas,

Thank you so much for your reply! I've already gotten the model that you provided. I'm trying to find the other macro model, and now I've found some models about sentinel. Once angain, thank you for your reply.

With kind regards,
Haonan Shi

**Comment by Namesume** [comment]
自动回复：您好，我已收到您的邮件。
